src/components/MnemonicVerification/MnemonicVerificationTs.ts

```diff
@@ -87,18 +86,41 @@ export class MnemonicVerificationTs extends Vue {
         const rebuilt = this.selectedWordIndexes.map((i) => this.shuffledWords[i]).join(' ');
 
         // - origin words list does not match
+        if (origin !== rebuilt) {
+            this.mnemonicCheckerNotification(origin, rebuilt);
```

Comment:
Dup codes. It can be moved before the check and keep only 1 call I guess

Reply:
Should be resolved